29.10.22/js/mathops.js

```diff
@@ -0,0 +1,12 @@
+const numberFirst = prompt('Input first number');
```

Comment:
You can combine methods
```suggestion
const numberFirst = Number(prompt('Input first number'));
```